geocity/apps/submissions/steps.py

```diff
@@ -83,6 +83,13 @@ def get_selectable_categories(submission=None, category_tags=None):
     ).distinct()
 
 
+def get_selectable_form(user, quick_access_slug):
+    forms = models.Form.objects.filter(quick_access_slug=quick_access_slug)
```

Comment:
Do we ensure a slug is unique and related to a unique form ?

Reply:
Yes, as can be seen in the diff of geocity/apps/forms/models.py, the slug column is marked `unique`